### ISSUE TITLE
findBWUsage.py script gets  "list index out of range [calculateBandwidth]" exception in error-logs.txt

### ISSUE THREAD

**Issue by mgonzcast** [litespeedtech/cyberpanel#1158]
I have reported this as a bug in the forum but I might think it´s better to do it here at github

I have Ubuntu 22.04. Everytime I run manually the findBWUsage.py script I get that exception logged in the error-logs.txt log.

I get the logs sents to me, so It´s full of those messages every hour. I have disabled the script in the crontab 

I have another server running Ubuntu 20.04 and it seems no exception is logged.

**Comment by usmannasir** [comment]
Did you by chance change how the web server logs are generated?

**Comment by mgonzcast** [comment]
> Did you by chance change how the web server logs are generated?

No, why? I just use the SMTP google option for sending emails. I get other emails just fine

**Comment by mgonzcast** [comment]
@usmannasir Have you had the chance to look at this?

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
